### PR TITLE
Update vmssPublicIpAddress.json

### PR DESCRIPTION
**Pull request by RajarajeshwariHS**
Updated the API version to the required one.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
Should API version also be updated in [vmssNetworkInterface.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssNetworkInterface.json)?

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json
Comment (sergey-shandar) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json]: Should it be `2019-06-01`?

Comment (RajarajeshwariHS) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json]: When I tried 2019-06-01, I get this error - 

**Get-AzureRmResource : NoRegisteredProviderFound : No registered resource provider found for location 'westus' and API version '2019-06-01' for type 'virtualMachineScaleSets/publicIPAddresses'. The supported api-versions are '2017-03-30, 2017-12-01, 2018-04-01, 2018-06-01, 2018-10-01'. The supported locations are 'eastus, eastus2, westus, centralus, northcentralus, southcentralus, northeurope, westeurope, eastasia, southeastasia, japaneast, japanwest, australiaeast, australiasoutheast, australiacentral, brazilsouth, southindia, centralindia, westindia, canadacentral, canadaeast, westus2, westcentralus, uksouth, ukwest, koreacentral, koreasouth, francecentral, southafricanorth, uaenorth'.**

May be I can chage it to '2018-10-01'

**Comment by sergey-shandar** [comment]
@MikhailTryakhov could you review the PR?

**Review by MikhailTryakhov — CHANGES_REQUESTED** [review]
No breaking changes currently applicable for this api version 2019-06-01 because we've merged and published it already...
So the nearest applicable date is around middle of September to update it.
so when branch would be ready we'll notify you and you'll be able to retarget

File: specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json
Comment (MikhailTryakhov) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json]: It's your choise which api version you need

Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json]: I think in this case you should use the latest one

**Comment by number213** [comment]
@RajarajeshwariHS please also update vmssNetworkInterface.json. It's latest API version is `2019-07-01` thought, I don't know if VMSS's NICs and PIPs should you different API versions in specs
```
type 'virtualMachineScaleSets/virtualMachines/networkInterfaces'. The supported api-versions are '2015-05-01-preview, 2015-06-15, 2016-03-30, 2016-04-30-preview, 2016-06-01, 2016-07-01, 2016-08-01, 2016-09-01, 2017-03-30, 2017-12-01, 2018-04-01, 2018-06-01, 2018-10-01, 2019-07-01'
```

**Comment by RajarajeshwariHS** [comment]
> @RajarajeshwariHS please also update vmssNetworkInterface.json. It's latest API version is 2019-07-01 thought, I don't know if VMSS's NICs and PIPs should you different API versions in specs
> type 'virtualMachineScaleSets/virtualMachines/networkInterfaces'. The supported api-versions are '2015-05-01-preview, 2015-06-15, 2016-03-30, 2016-04-30-preview, 2016-06-01, 2016-07-01, 2016-08-01, 2016-09-01, 2017-03-30, 2017-12-01, 2018-04-01, 2018-06-01, 2018-10-01, 2019-07-01'

**Comment by RajarajeshwariHS** [comment]
Also, I will make changes to vmssNetworkInterface.json as well. But, I will mention the API version as 2018-10-01 for both. As 2019-07-01 is not working for publicIPAddress because of the error above when I tried.

**Comment by RajarajeshwariHS** [comment]
I have done the required change here : https://github.com/Azure/azure-rest-api-specs/pull/6916

**Comment by RajarajeshwariHS** [comment]
> No breaking changes currently applicable for this api version 2019-06-01 because we've merged and published it already...
> So the nearest applicable date is around middle of September to update it.
> so when branch would be ready we'll notify you and you'll be able to retarget

Please let us know by mail. Thanks :)

**Review by anton-evseev — APPROVED** [review]

**Comment by number213** [comment]
@sergey-shandar gentle ping

**Review by anton-evseev — APPROVED** [review]

**Review by tjprescott — APPROVED** [review]

**Comment by number213** [comment]
@tjprescott CI's good

**Comment by tjprescott** [comment]
@number213 awaiting approval from @MikhailTryakhov. That's blocking the merge.

**Comment by MikhailTryakhov** [comment]
we do need to add this to the monthly branch, not to the master.
master contains 2019-06-01 and this change would be a breaking change. So please retarget and make correct change in 2019-07-01 folder

**Comment by number213** [comment]
Opened PR with this change into Network release branch: https://github.com/Azure/azure-rest-api-specs/pull/7008

**Pull request closed without merging**
